fiftyone/utils/depth_anything.py

```
@@ -0,0 +1,314 @@
+import fiftyone.core.labels as fol
+import fiftyone.core.utils as fou
+import fiftyone.utils.torch as fout
+import fiftyone.zoo.models as fozm
+
+fou.ensure_torch()
+import torch
+
+logger = logging.getLogger(__name__)
+
+
+def _ensure_depth_anything_3():
```

Ruff (0.14.10)

Missing return type annotation for private function `_ensure_depth_anything_3`

Add return type annotation: `None`

(ANN202)